proto/block.proto

```diff
@@ -0,0 +1,41 @@
+syntax = "proto3";
+
+import "digest.proto";
+
+
+// Block designated by a tag.
+enum BlockTag {
+    // The default value.
+    BLOCK_TAG_UNSPECIFIED = 0;
+    // The latest known block
+    BLOCK_TAG_LATEST = 1;
+}
+
+// Block uniquely designated by its height or a tag.
+message BlockSpec {
+    oneof spec {
+        // Specifies a block by its height.
+        uint64 block_number = 1;
+        // Specifies a block by a tag.
+        BlockTag block_tag = 2;
+        // Specifies a block by its hash.
+        // Digest block_hash = 3;
```

Comment:
commented out to stay with the request

https://github.com/0xPolygonMiden/miden-node/pull/9#discussion_r1253621985